tests/unit/OrgPerson.spec.ts

```diff
@@ -16,238 +13,289 @@ const vuetify = new Vuetify({})
-  existingCompletingParty: any
+  currentCompletingParty: any
 ): Wrapper<OrgPerson> {
   const localVue = createLocalVue()
   localVue.use(Vuetify)
   document.body.setAttribute('data-app', 'true')
+
   return mount(OrgPerson, {
     localVue,
-    propsData: {
-      currentOrgPerson: person,
-      activeIndex: activeIndex,
-      nextId: nextId,
-      existingCompletingParty: existingCompletingParty
-    },
+    propsData: { currentOrgPerson, activeIndex, nextId, currentCompletingParty },
     store,
     vuetify
   })
 }
 
-store.state.stateModel.nameRequest.entityType = 'BEN'
-store.state.stateModel.tombstone.currentDate = '2020-03-30'
-
 describe('Org/Person component', () => {
+  beforeAll(() => {
+    store.state.stateModel.nameRequest.entityType = 'BEN'
+    store.state.stateModel.tombstone.currentDate = '2020-03-30'
+  })
+
   it('Loads the component and sets data for person', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validPersonData, -1, 0, null)
+    const wrapper = createComponent(validPersonData, NaN, 0, null)
     await Vue.nextTick()
+
     expect(wrapper.vm.$data.orgPerson).toStrictEqual(validPersonData)
     expect((wrapper.vm as any).isIncorporator).toBe(false)
     expect((wrapper.vm as any).isDirector).toBe(true)
     expect((wrapper.vm as any).isCompletingParty).toBe(true)
+
     wrapper.destroy()
   })
 
   it('Loads the component and sets data for org', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, -1, 0, null)
+    const wrapper = createComponent(validOrgData, NaN, 0, null)
     await Vue.nextTick()
+
     expect(wrapper.vm.$data.orgPerson).toStrictEqual(validOrgData)
     expect((wrapper.vm as any).isIncorporator).toBe(true)
     expect((wrapper.vm as any).isDirector).toBe(false)
     expect((wrapper.vm as any).isCompletingParty).toBe(false)
+
     wrapper.destroy()
   })
 
-  xit('Displays form data for org', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, -1, 0, null)
-    expect((<HTMLInputElement> wrapper.find(orgNameSelector).element).value)
-      .toEqual(validOrgData['officer']['orgName'])
+  it('Displays form data for person', async () => {
+    const wrapper = createComponent(validPersonData, 0, null, null)
     await Vue.nextTick()
+
+    expect((wrapper.find(firstNameSelector).element as HTMLInputElement).value)
+      .toEqual(validPersonData['officer']['firstName'])
+    expect((wrapper.find(middleNameSelector).element as HTMLInputElement).value)
+      .toEqual(validPersonData['officer']['middleName'])
+    expect((wrapper.find(lastNameSelector).element as HTMLInputElement).value)
+      .toEqual(validPersonData['officer']['lastName'])
+
     expect(wrapper.find(doneButtonSelector).attributes('disabled')).toBeUndefined()
-    expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeDefined()
+    expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeUndefined()
```

Comment:
This is correct: if we edit an existing person then the Remove button is enabled.

(I moved the test from below to here. I think the diff is confused.)